### PR TITLE
init

### PR DESCRIPTION
**Pull request by modev0323**

### CODE REVIEW

File: .merlin [hunk 1]
```diff
@@ -1,13 +1,19 @@
 ####{BSB GENERATED: NO EDIT
```
Comment (vishakh) [inline .merlin]: Does this file have to be checked in? It contains some local configuration. Let's delete it if possible.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# mininax
+# mininax-reason
```
Comment (vishakh) [inline README.md]: Let's keep the project name as `mininax`.

File: package.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "mininax",
+  "name": "mininax-reason",
```
Comment (vishakh) [inline package.json]: Let's change the name back to `mininax`.

File: package.json [hunk 4]
```diff
@@ -13,16 +13,23 @@
   "keywords": [
     "BuckleScript"
   ],
-  "author": "Cryptonomic Inc",
-  "license": "GPL",
+  "author": "",
+  "license": "MIT",
```
Comment (vishakh) [inline package.json]: Please don't change the author and license.

File: package.json [hunk 5]
```diff
@@ -13,16 +13,23 @@
...
   "dependencies": {
+    "bs-conseiljs": "git+https://github.com/Cryptonomic/ConseilJS-ReasonML-Bindings.git#9d88c727b96fcebe437ea6b2d3819c1083927b74",
```
Comment (vishakh) [inline package.json]: This is fine for now but w should publish `bs-conseiljs` to NPM soon.

File: src/configs.re [hunk 6]
```diff
@@ -0,0 +1,16 @@
+let configs: array(Type.config) = [|
```
Comment (vishakh) [inline src/configs.re]: Can we remove this file from source control so these details are not public?

File: package.json (hunk 5 above)
Comment (anonymoussprocket) [reply package.json]: can it be part of the regular conseiljs package?

File: src/App.re [hunk 7]
```diff
@@ -0,0 +1,280 @@
...
+        </div>
+        <div className=Styles.mainContainer>
+          {switch (url.path) {
+            | [_, "accounts", _] => <Account items=state.account goToDetail={onSearchById} />
+            | [_, "operations", _] => <Operation items=state.operation goToDetail={onSearchById} />
+            | _ => <Block items=state.block goToDetail={onSearchById} goToBlock={getHashByLevel} />
+          }}
+        </div>
+        <Footer
+          searhVal={state.id}
```
Comment (anonymoussprocket) [inline src/App.re]: spelling

File: src/Error.re [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+[@react.component]
+let make = (~error, ~onTry) => {
+  let theme = React.useContext(ContextProvider.themeContext);
+  <div className=Styles.container onClick={_ => onTry()}>
+    <div className=Styles.mainContainer(theme)>
+      <div className=Styles.borderContainer>
+        <div className=Styles.contentContainer>
+          <div className=Styles.title>{ReasonReact.string("! ERROR !")}</div>
+          <p className=Styles.errTxt>{ReasonReact.string(error)}</p>
+          <button className=Styles.okBtn(theme)>{ReasonReact.string("Try Again")}</button>
```
Comment (anonymoussprocket) [inline src/Error.re]: Should we just extract the strings now in preparation for eventual localization? Even if no localization is added now.

File: package.json (hunk 5 above)
Comment (developer0623) [reply package.json]: No, we have to publish new npm dependency.

**Pull request merged**